Backend/AlgorithmsManager.py

```diff
@@ -474,20 +474,42 @@ def get_most_frequent_retweeters(self, username, num_tweets=20, num_retweets=20)
 
         return res
 
+
+
     def get_network_tweets_text(self, id2, username, count, depth):
         textsStr = ""
         if id2 > 0 or username is not None:
             # get bunch of tweets starting with the user of given id
             n = depth  # goes depth level deep
             # array of tweet texts
-            texts = []
-            self.append_texts_recursive(texts, count, n, 0, id2, username)
-
-            for text in texts:
-                splittext = text.split(' ')
-                for split in splittext:
-                    if not split.startswith('http'):
-                        textsStr += (split + " ")
+
+            #texts = []
+            # self.append_texts_recursive(texts, count, n, 0, id2, username)
+
+            # New network tweets method without using Recursive
+            depthDict = {}
+            depthDict[0] = [[id2, username]]
+
+            for i in range(0, depth+1):
+                for l in range(0, len(depthDict[i])):
```

Comment:
would you please rename this index variable to anything other than the letter 'l' (el)? Even with text highlighting from Github/IDE's, it's still confusing to read (i.e. looks like the number 1).